src/Controller/MainController.php

```diff
@@ -32,11 +32,9 @@ public function homeV2(
         VerbatimRepository $verbatimRepository,
         AuthenticationUtils $authenticationUtils
     ): Response {
-        if ($this->getUser()) {
-            return $this->redirectToRoute('re_user_redirectUser');
-        }
-
-        return $this->home($request, $verbatimRepository, $authenticationUtils);
+        return $this->getUser()
+            ? $this->redirectToRoute('re_user_redirectUser')
+            : $this->home($request, $verbatimRepository, $authenticationUtils);
```

Comment:
Il faudrait redirect to Route no n?

Reply:
On a actuellement `re_main_login` et `re_main_accueil` qui ont un path différent, mais utilisent la même fonction du controller `homeV2`.
Si on redirect to route, la redirection boucle à l'infini. Ça mérite une enquête sur un autre ticket qu'en penses tu ?